extensions/sql-database-projects/src/tools/netcoreTool.ts

```diff
@@ -35,17 +37,27 @@ export interface DotNetCommandOptions {
 export class NetCoreTool {
 
 	private static _outputChannel: vscode.OutputChannel = vscode.window.createOutputChannel(projectsOutputChannel);
+	private osPlatform: string = os.platform();
+	private netCoreSdkInstalledVersion: string | undefined;
+	private netCoreError: netCoreFindError = netCoreFindError.noError;
```

Comment:
`netCoreError` doensn't really seem accurate here. Something like `netcoreInstallState` would be better I think

Reply:
Updated this.